pset-functions.js

```diff
@@ -148,6 +294,58 @@ console.log('Problem 4:')
 
 // Add your code below this line
 
+
+// function consecutiveLetters(word){
+//
+//   const letters = word.split('')
+//
+//   console.log(letters)
+//
+//   for (let i = 1; i <= letters.length; i++) {
+//
+//     let currentLetter = letters[i-1]
+//     let compareLetter = letters[i]
+//
+//     console.log(currentLetter)
+//     console.log(compareLetter)
+//
+//     if (currentLetter === compareLetter){
+//       return true
+//     }
+//
+//   }
+//
+// return false
+
+// }
+
+// function to check if a word has conseuctive identical letters
+function consecutiveLetters(word){
+
+  console.log(word)
+
+// loop through every letter in the word, store the current letter and the letter after it in variables, and check each time through to see if they match.
+  for (let i = 1; i < word.length; i++) {
+
+    let currentLetter = word[i-1]
```

Comment:
Really great use of a for loop and taking advantage of the current iteration variable as your index!